src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/[[...slug]]/page.tsx

```diff
@@ -72,14 +72,12 @@ export default async function DashboardPage(props: Props) {
     email: session.user.email,
   });
 
-  const isNewUser = !isPrePlusUser(session.user);
```

Comment:
This bit is no longer relevant when Plus is disabled, so I removed it (cc @codemist).